pkg/network/usm/usm_http2_monitor_test.go

```diff
@@ -754,9 +754,9 @@ func (s *usmHTTP2Suite) TestRawTraffic() {
 		},
 		{
 			name: "validate http methods",
-			// The purpose of this test is to validate that we are not supporting http2 methods different from POST and GET.
+			// The purpose of this test is to validate that we are able to capture all http methods.
```

Comment:
let's add tests for Get and Post which are literals

Reply:
added